src/ansys/dpf/core/operators/geo/faces_area.py

```diff
@@ -0,0 +1,227 @@
+"""
+faces_area
+==========
+Autogenerated DPF operator classes.
+"""
+from warnings import warn
+from ansys.dpf.core.dpf_operator import Operator
+from ansys.dpf.core.inputs import Input, _Inputs
+from ansys.dpf.core.outputs import Output, _Outputs
+from ansys.dpf.core.operators.specification import PinSpecification, Specification
+
+
+class faces_area(Operator):
+    """Compute the measure of the Faces (surface for 2D faces of a 3D model
+    or length for 1D faces of a 2D model) using default shape
+    functions, except for polygons.
+
+    Parameters
+    ----------
+    mesh : MeshedRegion
+    mesh_scoping : Scoping
+        If not provided, the measure of all faces in
+        the mesh is computed. if provided,
+        the scoping needs to have "faces"
+        location.
```

Comment:
@MichaelNale @rafacanton same here, is the "the measure of all faces in the mesh" or "the measure of each face in the mesh"?